OpenRA.Renderer.Sdl2/VertexBuffer.cs

```diff
@@ -48,6 +48,16 @@ public void SetData(T[] data, int start, int length)
 			ErrorHandler.CheckGlError();
 		}
 
+		public void SetData(IntPtr data, int start, int length)
+		{
+			Bind();
```

Comment:
if we had one buffer for each renderer we would only need to set up the vertex format at initialization.